e2e/dotcms-e2e-node/frontend/utils/contentUtils.ts

```diff
@@ -75,7 +75,7 @@ export class ContentUtils {
       await dotIframe.getByRole("button", { name: " Import" }).click();
       await waitForVisibleAndCallback(
         dotIframe.getByRole("button", { name: " Remove" }),
-        async () => {},
+        async () => undefined,
```

Comment:
@victoralfaro-dotcms Passing a callback that is not utilized I think is not the better behavior. In this scenario, the callback should be optional, my suggestion:


```ts
export const waitForAndCallback = async (
  locator: Locator,
  state: "attached" | "detached" | "visible" | "hidden",
  callback?: () => Promise<void>,
): Promise<void> => {
  await waitFor(locator, state);
  if (callback) {
    await callback();
  }
};
```